.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
   # Run all tests before db_block_cache_test (db_test, db_test2)
   - JOB_NAME=unittests ROCKSDBTESTS_END=db_block_cache_test
   # Run all tests starting from db_block_cache_test (db_block_cache_test, ..., plain_table_db_test)
-  - JOB_NAME=unittests ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
+  - JOB_NAME=unittests MEM_ENV=true ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
```

Comment:
Similar way, checkpoint_test uses hardlink, which is hardly used in other places. db_wal_test uses read() instead of pread(), which is not used in other places.

I look at the tests. Maybe we can db_io_failure_test and keep corruption_test in MockEnv.